packages/clients/src/aptos/aptos.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2023-2024 Shinami Corp.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Aptos, AptosConfig } from "@aptos-labs/ts-sdk";
+
+const NODE_REST_URL = "https://api.shinami.com/aptos/node/v1";
+const NODE_INDEXER_URL =
+  "http://aptos-mainnet-graphql.aptos-node:8080/v1/graphql";
+
+/**
+ * Creates an Aptos client pointing to Shinami Node Service
+ * @param accessKey Node access key, set in Shinami UI. Note that this determines what network you targeting
+ * @param url Optional URL for fullnode. Defaults to Shinami's API
+ * @param indexerUrl Optional URL for indexer. Defaults to Aptos's mainnet public indexer.
+ * @returns Aptos client using Shinami Node service.
+ */
+export function createAptosClient(
+  accessKey: string,
+  url: string = NODE_REST_URL,
+  indexerUrl: string = NODE_INDEXER_URL,
+): Aptos {
+  return new Aptos(
+    new AptosConfig({
+      fullnode: url,
+      indexer: indexerUrl,
```

Comment:
Anything needed for `indexerConfig`?

Reply:
In the [docs](https://aptos-labs.github.io/aptos-ts-sdk/@aptos-labs/ts-sdk-1.12.2/classes/AptosConfig.html#indexerConfig), `indexerConfig` maps to just a series of headers we need to inject into the HTTP request for the request to succeed, identical to `clientConfig`.

I wasn't entirely sure if we needed to also add `indexerConfig` or if `clientConfig` was enough, so I added an extra test case in the aptos integration tests. And the answer to that question is no, we don't need `indexerConfig` to set API_KEY, `clientConfig` is enough.